deploy/examples/create-external-cluster-resources.py

```diff
@@ -1719,33 +1748,75 @@ def gen_json_out(self):
                     },
                 }
             )
-        if self.out_map["RBD_METADATA_EC_POOL_NAME"]:
+        # if 'RADOS_NAMESPACE' exists, then only add the "RADOS_NAMESPACE" namespace
+        if (
+            self.out_map["RADOS_NAMESPACE"]
+            and self.out_map["RESTRICTED_AUTH_PERMISSION"]
+        ):
             json_out.append(
                 {
-                    "name": "ceph-rbd",
-                    "kind": "StorageClass",
+                    "name": "rados-namespace",
+                    "kind": "Namespace",
```

Comment:
keep the Kind as `CephBlockPoolRadosNamespace` just Namespace is confusing

Reply:
Updated